composer.json

```diff
@@ -1,12 +1,16 @@
 {
-    "name": "bramus/ansi-php",
-    "description": "ANSI Control Functions and ANSI Control Sequences (Colors, Erasing, etc.) for PHP CLI Apps",
+    "name": "nahkampf/ansi-php",
+    "description": "ANSI Control Functions and ANSI Control Sequences (Colors, Erasing, etc.) for PHP CLI Apps. A fork of bramus/ansi-php.",
     "license": "MIT",
     "authors": [
         {
             "name": "Bramus Van Damme",
             "email": "bramus@bram.us",
             "homepage": "https://www.bram.us/"
+        },
+        {
+            "name": "Sebastian Utbult",
+            "email": "kreuzweg@gmail.com"
```

Comment:
If your intention is to actively keep working on this package, then I’ll happily accept you as a co-author instead of a regular contributor.

If not, then this section needs to be removed.